### PR TITLE
Adjust Swagger URLs to new `cow.fi` Domain

### PR DESCRIPTION
**Pull request by nlordell**
This PR adjusts the Swagger URLs to point to our new `cow.fi` domain! 🎉 🐮

I also decided to order the YAML pairs as `descritpion` -> `url` as I found it slightly more intuitive.

### Test Plan

Check it out! https://barn.api.cow.fi/docs/

image

### CODE REVIEW

File: crates/orderbook/openapi.yml [hunk 1]
```diff
@@ -3,20 +3,20 @@ info:
...
+  - description: Mainnet (Prod)
+    url: https://api.cow.fi/mainnet
+  - description: Rinkeby (Staging)
+    url: https://barn.api.cow.fi/rinkeby
+  - description: Rinkeby (Prod)
+    url: https://api.cow.fi/rinkeby
+  - description: xDai (Staging)
+    url: https://barn.api.cow.fi/xdai
+  - description: xDai (Prod)
+    url: https://api.cow.fi/xdai
```
Comment (MartinquaXD) [inline crates/orderbook/openapi.yml:17]: Shouldn't those refer to Gnosis chain?

Comment (nlordell) [reply crates/orderbook/openapi.yml]: Changed the description to mention `Gnosis Chain`. The URLs still use `xdai` as we haven't moved them over yet.

Comment (MartinquaXD) [reply crates/orderbook/openapi.yml]: Might be worth also updating the screen shot in the PR description.

Comment (nlordell) [reply crates/orderbook/openapi.yml]: 🦅 👀

**Review by MartinquaXD — APPROVED** [review]
Looks good to me.
One thing which surprises me is that the gnosis chain URLs still refer to xDai. Is there a discussion where one can read up about that decision? Could be confusing to only change the name but not add any new URLs with the new name. But this is besides the point of this PR.

**Comment by nlordell** [comment]
> Is there a discussion where one can read up about that decision?

Most of the discussion is happening in Slack. We've marked the `xdai` API endpoints as:
> Maybe later: Should we do at some point the rename to Gnosis Chain??

The issue is that the FE and our ingress configuration still use the `/xdai` path, so we need to change those before we can update the OpenAPI spec to use `/gnosischain` or `/gno` or `/gc` or whatever we settle on.

**Pull request merged**